### PR TITLE
Update Podfile.lock

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: components/TransactionsNavigationHeader.js [hunk 1]
```diff
@@ -26,7 +26,6 @@ export default class TransactionsNavigationHeader extends Component {
     Refill: 'refill',
     RefillWithBank: 'refillwithbank',
     RefillWithExternalWallet: 'qrcode',
-    Exchange: 'exchange',
```
Comment (Overtorment) [inline components/TransactionsNavigationHeader.js:29]: i see the menu entry being deleted.
but where is the link to `zigzag` itself? i suppose it should be deleted as well

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: components/TransactionsNavigationHeader.js (hunk 1 above)
Comment (marcosrdz) [reply components/TransactionsNavigationHeader.js:29]: My bad. forgot to save those changes. pushed

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/t7naestdch4qbpgcqpg2ul3z7u?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10346)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/aet2adztta33xlvzxtusjhhqgm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10347)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/cgopc7lhlltfianhymxjsatfwe?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10348)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**